clang/include/clang/CIR/Dialect/Builder/CIRBaseBuilder.h

```diff
@@ -350,13 +350,14 @@ class CIRBaseBuilderTy : public mlir::OpBuilder {
   }
 
   cir::StoreOp createStore(mlir::Location loc, mlir::Value val, mlir::Value dst,
-                           bool _volatile = false,
-                           ::mlir::IntegerAttr align = {},
+                           bool _volatile = false, uint64_t alignment = {},
                            cir::MemOrderAttr order = {}) {
     if (mlir::cast<cir::PointerType>(dst.getType()).getPointee() !=
         val.getType())
       dst = createPtrBitcast(dst, val.getType());
-    return create<cir::StoreOp>(loc, val, dst, _volatile, align, order,
+    const mlir::IntegerAttr alignmentAttr = mlir::IntegerAttr::get(
+        mlir::IntegerType::get(dst.getContext(), 64), alignment);
+    return create<cir::StoreOp>(loc, val, dst, _volatile, alignmentAttr, order,
```

Comment:
Is this right? If `alignment` is not given an alignment of 0 will be added to the store operation, which looks a bit suspicious.